### ISSUE TITLE
Single blending node for all

### ISSUE THREAD

**Issue by F-Burning** [opentoonz/opentoonz#4267]
When you want to blend nodes, you need to looking for between a lot of nodes

![o](https://user-images.githubusercontent.com/74098734/154974981-7ffa605c-c976-410b-b0a4-e9e8952328d5.jpg)

and this is specially slow when you use the search bar with ctrl + f and need to write the specific blending node that you need.

My proposal is create a single node where you can choose the blending option when you open the settings.

Something like this made fast

![01](https://user-images.githubusercontent.com/74098734/154977104-81be54e4-5ae5-45f2-bc62-e355289eb99f.png)

**Comment by oliver-dreamwalker** [comment]
I don't see how this can be faster. This is just moving the list of blending modes to another place, and what happens if you click the (?) button in this change ? is it show the help PDFs of all blending modes ?

**Comment by F-Burning** [comment]
Here is a better option


https://user-images.githubusercontent.com/74098734/155120432-259f6e60-731c-4232-b4d8-47de77f3eb2d.mp4




What do you think now? eh eh eh?

![210328d](https://user-images.githubusercontent.com/74098734/155116789-8e01fde1-e279-48a4-a9df-0cee48d19a5c.gif)

**Comment by oliver-dreamwalker** [comment]
Yeah, that's **WAY** better, but are you sure this will not conflict with other shortcuts ? By the way this change is VERY niche and only benefits people that use blending layers a lot, isn't better to add a **frequently used effects** list or a **favorite effects list** with a shortcut for easy/fast access ?

**Comment by F-Burning** [comment]
I think not, because this shortcuts only works in the schematic, and I think small details like this benefits us all when it comes to work. Being something that adds instead of subtracts I don't see the problem 😁The list is a good idea too, but it could be in addition to what I have commented, everything is to add facilities!

**Comment by artisteacher** [comment]
Personally I think it's a little limiting to have a shortcut just for quick access to the blend fx. I occasionally use the OverFx but otherwise don't really use any of the other blend FX. I actually use matte FX more than anything.  If I had quick access to any of the categories, I'd want it to be the matte FX. 

Rather than giving preference to a single category that not everyone uses, I'd also much rather see a shortcut to a **favorite fx list**. 

Alternatively, the different FX categories could be added to the "Configure Shortcut" panel so that users could add their own shortcuts for the categories they use the most. Though I still think the favorite FX access would be more helpful.

**Comment by F-Burning** [comment]
I think both you comment are a good addition too.

**Comment by RodneyBaker** [comment]
Converting this to Discussion so others can explore information and consider the best way forward.
